### ISSUE TITLE
Using '-1' as a version does not return the latest version

### ISSUE THREAD

**Issue by mattieserver** [Aiven-Open/karapace#668]
# What happened?
We trying to get the latest version of a schema with '-1' as version you get the following error:
```
URL: /subjects/XXXXXX/versions/-1
{"error_code":42202,"message":"The specified version '-1' is not a valid version id. Allowed values are between [1, 2^31-1] and the string \"latest\""}
```

# What did you expect to happen?

'-1' should work the same as using 'latest' as the version.

# What else do we need to know?

Docker image version: 3.6.1

**Comment by mattieserver** [comment]
If you look at https://github.com/aiven/karapace/blob/ec75a340d33a36c99eb5d107876fa08cb6ca38f9/karapace/schema_references.py#L79 it seems like it should have worked.

**Comment by nosahama** [comment]
Fixed in #885.
